sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/view/MeterSelector.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.view;
+
+import com.google.auto.value.AutoValue;
+import io.opentelemetry.sdk.metrics.internal.view.StringPredicates;
+import java.util.function.Predicate;
+import java.util.regex.Pattern;
+import javax.annotation.concurrent.Immutable;
+
+/**
+ * Provides means for selecting one or more Meters. Used for selecting instruments when constructing
+ * views.
+ */
+@AutoValue
+@Immutable
+public abstract class MeterSelector {
```

Comment:
I don't see test coverage for this class anywhere.

Reply:
Added